### PR TITLE
Resolve `Issue1741Suite` errors

### LINKED ISSUE #8763: Resolve `Issue1741Suite` errors
### Problem

We have the following tests suite `Issue1741Suite` that has problems when it's running against modularization codebase.

### Solution

Resolve issues with the test suite. When ready add @hapitest annotations to the passing tests.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
**Description**:

We have the following test suite Issue1741Suite that has problems when it's running against the modularization codebase.

**Solution**:

Resolve issues with the test suite. When ready add @hapitest annotations to the passing tests.

**Related issue(s)**:

Fixes #8763

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by netopyr — CHANGES_REQUESTED** [review]
The test currently fails because of our unstable test environment. I just ran the test with the fixes to the test environment applied and it succeeded without changes. Please wait with this PR until the test environment was fixed.

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/issues/Issue1741Suite.java [hunk 1]
```diff
@@ -49,7 +51,7 @@ public static HapiSpec queryPaymentTxnMustHavePayerBalanceForBothTransferFeeAndN
                 .then(getAccountInfo("payer")
                         .nodePayment(BALANCE)
                         .payingWith("payer")
-                        .hasAnswerOnlyPrecheck(INSUFFICIENT_PAYER_BALANCE));
+                        .hasCostAnswerPrecheck(INSUFFICIENT_PAYER_BALANCE));
```
Comment (netopyr) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/issues/Issue1741Suite.java:54]: This change is not needed.

Comment (natanasow) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/issues/Issue1741Suite.java]: Reverted.

**Review by netopyr — APPROVED** [review]
LGTM

**Pull request merged**